app/build.gradle

```diff
@@ -188,53 +190,56 @@ dependencies {
 
     annotationProcessor 'com.jakewharton:butterknife-compiler:8.0.1'
 
-    compileOnly 'com.google.auto.value:auto-value:1.5.2'
-    annotationProcessor 'com.google.auto.value:auto-value:1.5.2'
+    compileOnly "javax.annotation:jsr250-api:1.0"
     annotationProcessor 'com.ryanharter.auto.value:auto-value-parcel:0.2.5'
-    annotationProcessor 'com.ryanharter.auto.value:auto-value-moshi:0.4.0-rc2'
-    compileOnly 'com.ryanharter.auto.value:auto-value-moshi:0.4.0-rc2'
+    annotationProcessor 'com.ryanharter.auto.value:auto-value-moshi:0.4.0'
+    compileOnly 'com.ryanharter.auto.value:auto-value-moshi:0.4.0'
 
     internalDebugImplementation 'com.jakewharton:process-phoenix:1.0.2'
 
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.4'
-    internalReleaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.4'
-    productionReleaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.4'
-    practiceReleaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.4'
+    def leakCanaryVersion = "1.6.2"
```

Comment:
I'd be fine w/ ripping all the debug related libraries out, not necessary now, but might make things easier later